### PR TITLE
replace local subworkflow with nf-core version - vcf_annotate_ensemblvep

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by matthdsm — APPROVED** [review]

File: modules.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
                     },
                     "bcftools/mpileup": {
                         "branch": "master",
-                        "git_sha": "6301e29d77e7ec7ce98b55b8a361b316a9a91bfe"
+                        "git_sha": "6301e29d77e7ec7ce98b55b8a361b316a9a91bfe",
```
Comment (FriederikeHanssen) [inline modules.json:15]: why is it mpileup checksum updated?

**Review by FriederikeHanssen — CHANGES_REQUESTED** [review]

File: subworkflows/nf-core/vcf_annotate_ensemblvep/main.nf [hunk 2]
```diff
@@ -22,13 +22,13 @@ workflow VCF_ANNOTATE_ENSEMBLVEP {
     TABIX_BGZIPTABIX(ENSEMBLVEP.out.vcf)
 
     // Gather versions of all tools used
-    ch_versions = ch_versions.mix(ENSEMBLVEP.out.versions.first())
-    ch_versions = ch_versions.mix(TABIX_BGZIPTABIX.out.versions.first())
```
Comment (FriederikeHanssen) [inline subworkflows/nf-core/vcf_annotate_ensemblvep/main.nf:26]: ok so cleared up now to not use `.first()`?

File: subworkflows/local/vcf_annotate_all/main.nf [hunk 3]
```diff
@@ -2,9 +2,9 @@
 // ANNOTATION
 //
 
-include { VCF_ANNOTATE_ENSEMBLVEP                       } from '../vcf_annotate_ensemblvep/main'
-include { VCF_ANNOTATE_ENSEMBLVEP as VCF_ANNOTATE_MERGE } from '../vcf_annotate_ensemblvep/main'
-include { VCF_ANNOTATE_SNPEFF                           } from '../../nf-core/vcf_annotate_snpeff/main'
+include { VCF_ANNOTATE_ENSEMBLVEP                       } from '../../nf-core/vcf_annotate_ensemblvep/main.nf'
+include { VCF_ANNOTATE_ENSEMBLVEP as VCF_ANNOTATE_MERGE } from '../../nf-core/vcf_annotate_ensemblvep/main.nf'
```
Comment (FriederikeHanssen) [inline subworkflows/local/vcf_annotate_all/main.nf:6]: can we continue to import without `.nf` for consistency sake

**Comment by maxulysse** [comment]
@nf-core-bot fix linting pretty please :pray:

File: modules.json [hunk 4]
```diff
@@ -8,420 +8,595 @@
                     "ascat": {
                         "branch": "master",
                         "git_sha": "5e34754d42cd2d5d248ca8673c0a53cdf5624905",
-                        "installed_by": ["modules"]
+                        "installed_by": [
+                            "modules"
```
Comment (FriederikeHanssen) [inline modules.json]: prettier?

**Comment by FriederikeHanssen** [comment]
the annotate.conf probably needs someupdating too since we rely on the naming there

**Comment by maxulysse** [comment]
> the annotate.conf probably needs someupdating too since we rely on the naming there

Was planning to do so in a coming commit

**Pull request merged**